include/pmacc/math/complex/Complex.tpp

```diff
@@ -277,12 +278,21 @@ namespace pmacc
 
     namespace mpi
     {
-        using complex_X = pmacc::math::Complex<picongpu::float_X>;
+        // Specialize complex type grid buffer for MPI
+        template<>
+        HINLINE MPI_StructAsArray getMPI_StructAsArray<pmacc::math::Complex<float>>()
+        {
+            using complex_X = pmacc::math::Complex<float>;
```

Comment:
Maybe this type should be called `Complex` ? Same later

Reply:
I will change it because I need to touch this PR because of the other suggestion anyway.